lib/contributions/index.js

```diff
@@ -114,22 +114,27 @@ var GenerateRepository = module.exports = function (options, done, progress) {
                 if (err) { return done(err); }
                 repo.commit("Added the title and description (/cc. @IonicaBizau)", function (err) {
                     if (err) { return done(err); }
-                    var output = Fs.createWriteStream(repoPath + ".zip")
-                      , archive = Archiver("zip")
-                      , zipErr = null
-                      ;
-
-                    output.on("close", function () {
-                        done(zipErr, repoId);
-                    });
-
-                    archive.on("error", function(err){
-                        zipErr = err;
-                    });
-
-                    archive.pipe(output);
-                    archive.directory(repoPath, "source");
-                    archive.finalize();
+
```

Comment:
Just delete this commented code. If we will need it, we will just take it from the git history.